frontend/javascript/code_snippets.js

```diff
@@ -3,6 +3,70 @@ export function enableCodeHighlighting() {
   hljs.highlightAll();
 }
 
+const h1Elements = document.querySelectorAll('h1');
```

Comment:
this code needs to be encapsulated inside of a function that gets invoked from `index.js`